packages/ui/primitives/avatar.tsx

```diff
@@ -75,7 +76,7 @@ const AvatarWithText = ({
     </Avatar>
 
     <div
-      className={cn('hidden flex-col text-left text-sm font-normal lg:flex', showText && 'flex')}
+      className={cn('hidden flex-col text-left text-sm font-normal lg:flex', textSectionClassName)}
```

Comment:
Why is hidden still here? Get rid of the revert lg:flex and hidden

It's causing text top not be visible in the actual switcher

<img width="552" alt="image" src="https://github.com/documenso/documenso/assets/20962767/1934c987-c1c2-482c-b833-5f7ac4fdbd81">

Also if the design wants the avatars to be aligned try ml-3

Reply:
@adithyaakrishna can you please link me the designs

Reply:
@dguyen Here's the link, https://www.figma.com/file/k44S7KkjaT16S88n6BVjx7/Single-Player-Mobile?type=design&node-id=2614%3A1805&mode=design&t=tdgZ7CnLCj2um4uX-1

Reply:
@adithyaakrishna could you please compare the image I posted and the design you linked?

The text next to the avatars within the dialog is not visible. You have inadvertently broken all usage of this component on mobile due to hardcoding this hidden class onto the avatar.

I asked you to add a class prop for the avatar text section so you can hide it only for the header.

Reply:
@dguyen My bad, I missed the text next to the Avatars on the dropdown. I've added the classProp and also removed the hardcoded className